src/_pytest/outcomes.py

```diff
@@ -49,18 +49,23 @@ class Failed(OutcomeException):
 class Exit(KeyboardInterrupt):
     """ raised for immediate program exits (no tracebacks/summaries)"""
 
-    def __init__(self, msg="unknown reason"):
+    def __init__(self, returncode=None, msg="unknown reason"):
         self.msg = msg
+        self.returncode = returncode
         KeyboardInterrupt.__init__(self, msg)
 
 
 # exposed helper methods
 
 
-def exit(msg):
-    """ exit testing process as if KeyboardInterrupt was triggered. """
+def exit(msg, returncode=None):
```

Comment:
Do you suggest a different default value @nicoddemus ? I thought in using 1 or importing `EXIT_INTERRUPTED` from _pytest/main.py but I wasn't sure.

Reply:
Let's keep `None` and handle that in `main`. 👍